internal/locate/region_cache.go

```diff
@@ -2629,6 +2649,195 @@ func invokeKVStatusAPI(addr string, timeout time.Duration) (l livenessState) {
 	return
 }
 
+const (
+	slowScoreInitVal     = 1
+	slowScoreThreshold   = 80
+	slowScoreMax         = 100
+	slowScoreInitTimeout = 500000   // unit: us
+	slowScoreMaxTimeout  = 30000000 // max timeout of one txn, unit: us
```

Comment:
```suggestion
        slowScoreInitTimeoutInUs = 500000 // unit: us
	slowScoreMaxTimeoutInUs  = 30000000 // max timeout of one txn, unit: us
```
Or use duration instead.

Reply:
Is this for all read queries? For large scan query,  30s may be the legitimate latency.

Reply:
> legitimate

Yes, 30s is the largest and tolerable TIMEOUT setting used in Client by default.